public/locales/en/translation.json

```diff
@@ -127,13 +126,13 @@
     "gasCost": "Gas cost",
     "gasFreeTrade": "Gasless swap",
     "insufficientBalance": "Insufficient {{symbol}} balance",
+    "makeNewOrder": "Make new order",
     "moreInformation": "More information",
     "nativeCurrencyMaxInfoText": "{{amount}}. This amount is calculated after subtracting a {{fee}} gas fee from your total balance.",
     "newQuoteIn": "New Quote In",
-    "newSwap": "New swap",
-    "noIndexersFound": "No indexer nodes found",
     "noIndexerOrdersFound": "No orders found for this token pair",
-    "orderSuccessfullyCreated": "Order successfully created",
+    "noIndexersFound": "No indexer nodes found",
+    "orderSuccessfullyCreated": "Order successfully created.",
```

Comment:
this line has a period at the end but most other lines don't. Will that create inconsistent UX?